### DISCUSSION TITLE
Triggered Cloudflare bot protection

### DISCUSSION THREAD

**Discussion by Jhon-Wilmer-JK** [crunchy-labs/crunchy-cli#409] [Q&A]
![image](https://github.com/crunchy-labs/crunchy-cli/assets/123090106/fa76b0ea-ee2c-4e5c-a440-06d84357f1b2)
Does anyone know how to avoid this problem on VPS?
It only happens on VPS but it works without problems on my PC

**Comment by Frooastside** [comment]
Duplicate of #104

Cloudflare flags your vps as suspicous because it knows the ip belongs to a hosting company and crunchyroll users dont usually sit in datacenters. you can try to use a vpn or proxy or maybe using a custom user agent from the switch can help: `--user-agent "Crunchyroll/1.7.0 Nintendo Switch/12.3.12.0 UE4/4.27"`
